### DISCUSSION TITLE
Twitter the official?

### DISCUSSION THREAD

**Discussion by telmobss1** [lxqt/lxqt#1949] [Q&A] [answered]
A question arose here, and as there is no documentation I decided to question

Is this Twitter account official?

And who manages the account?

https://mobile.twitter.com/LXQtOfficial?s=09

**Comment by stefonarch** [comment]
> And who manages the account?

Found it here https://github.com/lxqt/lxqt/issues/1490#issuecomment-390446909
@tsimonq2

**Reply by telmobss1** [reply]
I would like to call @agaida I would like to call @agaida I would like to call @agaida I would like to call @agaida it's the @tsimonq2 If they are available to chat

**Reply by tsujan** [reply]
You're very late to the show ;)

**Reply by telmobss1** [reply]
I do not understand

**Reply by stefonarch** [reply]
Short story: agaida disappeared, long story https://github.com/lxqt/lxqt/issues/1829

**Reply by telmobss1** [reply]
and the account name and password?Lost if?

**Reply by stefonarch** [reply]
As you can read in the issue  there was much more important stuff lost with him.

**Comment by telmobss1** [comment]
If no one knows the user's account or password, then the best thing to do is to create a new Twitter account

Then share the username and password to everyone who is directly involved with the lxqt project, because if it disappears, they can pass on Twitter to someone else


In short, this is what I call free software

**Reply by stefonarch** [reply]
We should first email @tsimonq2, and we could also claim the account directly from twitter. As it says "official" there has to be only one.

**Reply by wxl** [reply]
I can probably track down Simon. He's been ridiculously busy lately and has more or less checked out because of it. I have my ways…

**Reply by telmobss1** [reply]
ok, i thought it was abandoned





doned, in this case can you talk to him?

That way, I could take care of social media

**Reply by stefonarch** [reply]
Forgot about this discussion here, it's now some duplicate of https://github.com/lxqt/lxqt/issues/2070#issuecomment-915130585
Anyway, at the moment we have https://twitter.com/lxqt_project and I'll try  for a third time to recover our  @lxqtofficial

@telmobss1  you're  still interested?

**Comment by stefonarch** [comment, marked as answer]
https://twitter.com/LXQtOfficial/status/1532102153606770688

So soon there will be only https://twitter.com/lxqt_project

**Reply by yan12125** [reply]
By "deactivated", wil tweets on https://twitter.com/LXQtOfficial removed? If so, can we preserve them somewhere? Maybe this works: https://yourbusiness.azcentral.com/copy-old-tweets-onto-new-account-23492.html

**Reply by stefonarch** [reply]
It was my idea, but it could be changed. But having two accounts is not good too.

I'm not sure about the c&p: tweets from 2018 will be new ones afaik,  most of them are retweets.

**Reply by yan12125** [reply]
Never mind, here is a simpler way! https://web.archive.org/web/20220602052920/https://twitter.com/LXQtOfficial/

PS. Yes, keeping two accounts is not good.
